requirements/base.txt

```diff
@@ -8,13 +8,13 @@ appdirs==1.4.4
     # via fs
 asgiref==3.8.1
     # via django
-django==4.2.20
+django==5.2
```

Comment:
It should be supporting PR. You are actually upgrading to 52. Please maintain the 42.